include/RAJA/policy/cuda/kernel/CudaKernel.hpp

```diff
@@ -87,7 +89,7 @@ namespace statement
  */
 template <typename LaunchConfig, typename... EnclosedStmts>
 struct CudaKernelExt
-    : public internal::Statement<cuda_exec<0>, EnclosedStmts...> {
+    : public internal::Statement<cuda_exec_explicit<0, 0>, EnclosedStmts...> {
```

Comment:
I think this should this be using LaunchConfig instead of cuda_exec. cuda_exec isn't the right kind of policy for kernel.

Reply:
No, `cuda_exec_explicit` is correct here. The `Statement` itself requires an execution policy, while the `LaunchConfig` is a specialization depending on the `cuda_launch` type.

Reply:
Ok, I see what you mean, the statement needs basically a placeholder cuda policy, the exact type or args do not matter. Can you add something like that in the comment?

Reply:
Done!